docs/branding.md

```diff
@@ -19,3 +19,20 @@ You can change the logo shown on the CMS login screen using the `login-logo` con
 	}
 }
 ```
+
+## Custom admin colour scheme
+
+To register a custom admin colour scheme add the following function on `admin_init` action.
+
+```php
+wp_admin_css_color( string $key, string $name, string $url, array $colors = [], array $icons = [] );
+```
+
+- `key`: The unique key for this theme.
+- `name`: The name of the theme.
+- `url`: The URL of the CSS file containing the color scheme.
+- `colors`: An array of CSS color definition strings which are used to give the user a feel for the theme.
+- `icons`: CSS color definitions used to color any SVG icons.
+  - `base`: SVG icon base color.
+  - `focus`: SVG icon color on focus.
+  - `current`: SVG icon color of current admin menu link.
```

Comment:
Can you add an example of this too, including the `admin_init` action hook? More documentation is better and this doesn't make it clear that you need to use hexadecimal or CSS friendly colour names.